scripts/kvm-test.py

```diff
@@ -252,6 +260,9 @@ def parse_args():
                         'LIVEFS_EDITOR to it\n'
                         'https://github.com/mwhudson/livefs-editor')
 
+    if ctx.args.bios and ctx.args.secureboot:
+        raise Exception("--secure-boot and --bios are mutually exclusive")
```

Comment:
Consider using argparse's native handling of exclusive groups?

```diff
- parser.add_argument('-B', '--bios', action='store_true', default=False,
-                    help='boot in BIOS mode (default mode is UEFI)')
- parser.add_argument('--secure-boot', action='store_true', default=False,
-                     help='Use SecureBoot', dest="secureboot")
+ boot_group = parser.add_mutually_exclusive_group()
+ boot_group.add_argument('-B', '--bios', action='store_true', default=False,
+                        help='boot in BIOS mode (default mode is UEFI)')
+ boot_group.add_argument('--secure-boot', action='store_true', default=False,
+                         help='Use SecureBoot', dest="secureboot")

Reply:
TIL another thing about argparse. Thanks for this!